src/corecel/io/ScopedStreamRedirect.hh

```diff
@@ -35,7 +35,7 @@ class ScopedStreamRedirect
 {
   public:
     // Whether stream redirection is enabled
-    static bool allow_redirect();
+    static bool enable_redirect();
```

Comment:
Both the old and new spelling conflict (for me) with the documentation.
They both sounds to me as "please turn-on/enable redirection starting now (if not already on)'
as opposed to asking the question "is it already enabled?"
```suggestion
    static bool redirect_is_enabled();
```
The comment applies equally to `enable_profiling` and `enable_signals`.

Reply:
Since it's a static method, it would be `ScopedProfiling::enabled()` etc.

Reply:
I agree that it is something that does not 'work' for us but it (unfortunately) compiles: https://godbolt.org/z/G87xq5a5f

Reply:
Ah :) ... that is 'just' the `this` keyword not being defined in a static function.  The version that compiles (https://godbolt.org/z/jYrxbKqYq) is as unclear of what is 'enabled' :)